charts/descheduler/templates/cronjob.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
 spec:
   schedule: {{ .Values.schedule | quote }}
   concurrencyPolicy: "Forbid"
+  {{- if .Values.priorityClassName }}
```

Comment:
I think this should probably be `{{- if .Values.startingDeadlineSeconds }}

Reply:
Correct. I amended the commit to fix this